swagger.yml

```diff
@@ -1360,6 +1360,11 @@ definitions:
           - US_WEST_1
           - US_WEST_2
           - US_EAST_1
```

Comment:
should add `US_EAST_2`

Reply:
nit - it would be helpful to mention `volumeType` is only used in `createCluster`

Reply:
it also shows up when getting clusters

Reply:
when creating clusters, this is only used when the cluster is single-region